Jenkins_setup.md

```diff
@@ -142,6 +146,52 @@ The teardown script is entered into the Jenkins Post-Build Actions dialog like:
 
 ![Jenkins Build dialog](pictures/jenkins_post_build.png)
 
+## Setting up a bare metal Jenkins Slave Node
+
+### Setting up a jenkins user
+
+Generally, Jenkins Master needs to log into the slave node as `jenkins` user to run the
+all Jenkins job.
+So you need to create a new user `jenkins` on your slave node and add this new user to
+the `sudo` group, which grants it `sudo` priviledges.
+
+```
+$ sudo adduser jenkins
+$ sudo usermod -aG sudo jenkins
+```
+
+Not only just having sudo priviledges, the `jenkins` user also needs to be able to run `sudo`
+without a password.
+Log into your slave node as root and open the `/etc/sudoers` file with `visudo` editor.
+> **Note** You should never edit `/etc/sudoers` with a regular text editor, such as `Vim`
+> or `nano`, because they do not validate the syntax like the `visudo` editor.
+```
+sudo visudo
```

Comment:
Can you add a `$ ` shell prefix? See https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#code-blocks.